### DISCUSSION TITLE
How to set a size of an area?

### DISCUSSION THREAD

**Discussion by XANi** [pterm/pterm#564] [Q&A]
Is there a way to set the size of the Area ? I'd like to have one that doesn't change size based on the text inside so the layout doesn't flicker when data changes.

I basically have 2 areas within a panel next to eachother and I'd like to make the left one be the constant size regardless of content.

**Comment by MarvinJWendt** [comment]
Hi @XANi, this is currently not possible, sadly. Feel free to make a feature request in the issues :)
